dataset/jeopardy_dataset.py

```diff
@@ -0,0 +1,275 @@
+import os.path
+import json
+from PIL import Image
+from torch.utils.data import Dataset
+import torch
+import os
+import string
+from tqdm import tqdm
+import nltk
+
+# START_TOKEN = "<s>"
+# END_TOKEN = "</s>"
+PAD_TOKEN = "<pad>"
```

Comment:
I don't think we need to specify these given the roberta tokenizer anymore? Delete if so